pkg/config/mesh/mesh.go

```diff
@@ -220,6 +222,8 @@ func ApplyMeshConfig(yaml string, defaultConfig meshconfig.MeshConfig) (*meshcon
 		}
 	}
 
+	defaultConfig.TrustDomainAliases = sets.NewSet(append(defaultConfig.TrustDomainAliases, prevTrustDomainAliases...)...).SortedList()
```

Comment:
What about the other fields? Like CaCertificates DiscoverySelectors

Reply:
Since we are selecting default value for a field, can that be done in a separate PR?

Reply:
make sense